tests/integration/routes/test_dump.py

```diff
@@ -136,7 +136,12 @@ def test_dump_submission_authenticated_with_role_with_answers(self):
                     'exercise_sid': '789',
                     'schema_name': 'test_radio_mandatory',
                 },
-                'data': [{'answer_id': 'radio-mandatory-answer', 'value': 'Coffee'}],
+                'data': {
+                    'answers': [
+                        {'answer_id': 'radio-mandatory-answer', 'value': 'Coffee'}
+                    ],
+                    'lists': {},
```

Comment:
I don't see any tests we now test the new format with list dict populated

Reply:
We've added extra test for it